cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The `include` function is used with a `namespace` argument, but the `cinema.urls` and `user.urls` modules must define an `app_name` attribute for namespaces to work correctly. Ensure that both `cinema.urls` and `user.urls` have `app_name` defined.